mpa/modules/ov/graph/parsers/cls/cls_base_parser.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from typing import Dict, List, Optional
+
+from mpa.utils.logger import get_logger
+
+from ..builder import PARSERS
+from ..parser import parameter_parser
+
+
+logger = get_logger()
+
+
+@PARSERS.register()
+def cls_base_parser(
+    graph, component: str = "backbone"
+) -> Optional[Dict[str, List[str]]]:
+    assert component in ["backbone", "neck", "head"]
+
+    possible_neck_input_types = ["ReduceMean", "MaxPool", "AvgPool"]
```

Comment:
Isn't it good to move this to global variable at the top of the code?

Reply:
Fixed in https://github.com/openvinotoolkit/model_preparation_algorithm/pull/95/commits/e173066e7d7a43670452fb1b3dc9e99690c4503a